package.json

```
@@ -34,10 +34,10 @@
     "lint": "eslint .",
     "build": "babel src/ --out-dir build --ignore '**/*.test.js'",
     "test": "jest"
-  },
+  }, 
   "peerDependencies": {
-    "react": "^16.9.0",
-    "react-native": "^0.61.5"
+    "react": "~16.8.6 || ~16.9.0 || ~16.11.0 || ~16.13.1 || ~17.0.1",
+    "react-native": "^0.60.6 || ^0.61.5 || ^0.62.2 || ^0.63.2 || ^0.64.0 || 1000.0.0"
```

maybe we can do this
```suggestion
    "react": ">=16.8.0",
    "react-native": ">=0.60.0"
```

---

yes, I think that should work too

---

Just to be on the safe side
```suggestion
    "react": ">=16.9.0",
    "react-native": ">=0.61.5"
```

---

Can you commit these suggestions so we can go ahead and release your PR?